blog/content/post/2020-04-16-remote_work_post.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Remote productivity
+abstract: 'About remote productivity in 3mdeb.'
```

Comment:
@KBanecka this should be something that post as pitch on social media, it seems to be little bit too short and not very encouraging.

Reply:
Ok, I'll work it out

Reply:
[commit](https://github.com/3mdeb/news-and-ideas/commit/dd554d2ecb716dae3e9e9afe9585bc912cd927c1#diff-a68390b5dcec9ebb7b7e5492bfbe6cdc)